src/common/types.ts

```diff
@@ -223,6 +224,21 @@ export type ServerInfoType = 'MemberCount';
 export type RoleTypeKey = keyof typeof RoleType;
 export type Maybe<T> = T | undefined | null;
 
+export type MessageSendData =
+  | string
+  | discord.MessagePayload
+  | (discord.ReplyMessageOptions & { split?: false }); // Discord v13 change
```

Comment:
can this go in a smaller change since it's a v13 thing?

Reply:
Sure